forum_leaderboard/static/html/forum_leaderboard_studio.html

```diff
@@ -0,0 +1,10 @@
+<div class="forum_leaderboard_block">
+<h2>{{ display_name }}</h2>
```

Comment:
In my testing, `display_name` is not getting passed to the template here, so this `<h2>` is empty and it results in a weird blank space at the top of the xblock.